test/kwtool/crawler/keyword_results_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule Kwtool.KeywordResultsTest do
+  use Kwtool.DataCase, async: true
+
+  alias Kwtool.Crawler.KeywordResults
+
+  describe "search/2" do
+    test "given an existing keyword, returns the matching keyword result" do
+      user = insert(:user)
+      keyword = insert(:keyword, phrase: "coffee", status: "finished", user: user)
+      insert(:keyword_result, keyword: keyword)
+
+      [keyword_result] = KeywordResults.search(user, %{keyword: "coffee"})
+
+      assert Enum.count(keyword_result.keyword_results) == 1
+      assert keyword.id == keyword_result.id
+      assert keyword.phrase == "coffee"
```

Comment:
I'm not sure about this assertion, seems we're testing the record we created above. maybe we should

```suggestion
      assert keyword_result.id == keyword.id
      assert keyword_result.keyword.phrase == "coffee"
```

What do you think?

Reply:
yeah, right, here the naming is quite confusing, I updated it to `expected_keyword` it could be clearer. https://github.com/byhbt/kwtool/commit/2bb53e4849f37561e61a1d711c1d7ba321f827f7